pkg/backend/httpstate/backend.go

```diff
@@ -1395,6 +1450,7 @@ func (b *cloudBackend) createAndStartUpdate(
 	}
 	// Check if the user's org (stack's owner) has Copilot enabled. If not, we don't show the link to Copilot.
 	isCopilotEnabled := updateDetails.IsCopilotIntegrationEnabled
+	b.copilotEnabledForCurrentProject = &isCopilotEnabled
```

Comment:
I'm still very confused about this flag and how we set it at `createAndStartUpdate` as opposed to all other properties of `cloudBackend`. I don't have a great solution but I hope we'll clean this up with the proposed functionality capabilities check.

Reply:
Yes, if we could do the capabilities check before starting things off we could remove all this stuff.